client/src/pages/AboutUs.js

```diff
@@ -377,6 +379,25 @@ const AboutUs = () => {
         </SupportersLogosContainer>
       </SupporterContainer>
 
+      <PressContainer>
+        <FlexBox direction="column" align="center">
+          <h1>{t("Press")}</h1>
+          <p>{t("If you're a member of the press, contact us at ")}</p>
```

Comment:
For translation, you will need to add keys yo `client/src/locales/translations/en_US.json`
(For this particular case inside "aboutUs") like what @wadehrarshpreet said